modules/gdscript/editor/script_templates/CharacterBody3D/godot_character_controller.gd

```diff
@@ -0,0 +1,73 @@
+# meta-description: Godot character controller for gravity games (FPS, TPS, ...)
+
+extends _BASE_
+
+
+@export_group("Kinematic")
+@export var walk_max_acceleration := 300.0
+@export var walk_acceleration := 15.0
+@export var run_max_acceleration := walk_max_acceleration * 1.5
+@export var run_acceleration := walk_acceleration * 1.5
+@export var friction := 200.0
+@export var jump_velocity := 5.0
+@export_group("Camera")
+@export var mouse_sensitivity := 0.005
+@export var camera_max_vertical_angle := 30
+@export var camera_min_vertical_angle := -10
+
+# Get the global gravity from the project settings. RigidBody3D nodes also use this value.
+var gravity: float = ProjectSettings.get_setting("physics/3d/default_gravity")
+var _current_acceleration := 0.0
+
+# Nodes for camera management, replace these nodes by yours and remove the initialization in _on_ready.
+var spring_arm := SpringArm3D.new()
+var camera := Camera3D.new()
+
+func _unhandled_input(event):
+	# The mouse cursor is not visible in the game, you can toogle the behavior with the escape key.
+	if event.is_action_pressed("ui_cancel"):
+		Input.set_mouse_mode(Input.MOUSE_MODE_CAPTURED if Input.get_mouse_mode() != Input.MOUSE_MODE_CAPTURED else Input.MOUSE_MODE_VISIBLE)
+
+	# Moves the camera according to the movement of the cursor.
+	elif Input.get_mouse_mode() == Input.MOUSE_MODE_CAPTURED and event is InputEventMouseMotion:
+		rotate_y(-event.relative.x * mouse_sensitivity)
+		spring_arm.rotate_x(-event.relative.y * mouse_sensitivity)
+		spring_arm.rotation.x = clamp(spring_arm.rotation.x, deg_to_rad(camera_min_vertical_angle), deg_to_rad(camera_max_vertical_angle))
+
+func _ready() -> void:
+	Input.set_mouse_mode(Input.MOUSE_MODE_CAPTURED)
+	spring_arm.position = Vector3(0, 1, 0)
```

Comment:
I had done it at first, then I had deleted it because when you replace for your own camera (which will be ultimately the case), these properties will become useless.